cpp/include/cuspatial/point_in_polygon.hpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <memory>
+#include <cudf/column/column.hpp>
+#include <cudf/column/column_view.hpp>
+#include "cudf/types.hpp"
+
+namespace cuspatial {
+
+/**
+ * @brief Point-in-Polygon (PIP) tests between a column of points and a
+ *        column of polygons
+ *
+ * Performs collision detection for each test point against at most 32 polygons. Polygons are
```

Comment:
This is not collision detection.
```suggestion
 * Tests whether points are inside at most 32 polygons. Polygons are
```